e2e/tests/components/sliders/sliders.po.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { browser, element, by } from 'protractor';
+
+export class SlidersPage {
+        
+    getPage(): void {
+        browser.get('/sliders');
+    }
+    
+    titleText = browser.getTitle();
+    
+    slider1 = element(by.id('slider1'));
+    slider2 = element(by.id('slider2'));
+    slider3 = element(by.id('slider3'));
+    slider4 = element(by.id('slider4'));
+    slider5 = element(by.id('slider5'));
+    slider6 = element(by.id('slider6'));
+    slider7 = element(by.id('slider7'));
+    input1 = element(by.id('input1'));
+    input2 = element(by.id('input2'));
+    
+    getHandleAttribute = function(slider: any, handle: string, attribute: string) {

```

Comment:
class functions in TypeScript should be written without the function keyword, e.g.:

```typescript
getHandleAttribute(slider: any, handle: string, attribute: string) {
    ...
}
```